core-web/libs/dotcms-scss/angular/dotcms-theme/components/_image.scss

```diff
@@ -0,0 +1,43 @@
+//TODO: implement image styles
```

Comment:
same galleria.scss

Reply:
it si now, but soon will work on that in a sub-tasks of the same feature.

Reply:
So this is not getting merged?

Reply:
yes will be merged, just with all commented to don't look for the styles template/reference when working on in in this same sprint.